presto-hive/src/test/java/com/facebook/presto/hive/TestAriaHiveDistributedQueries.java

```diff
@@ -240,6 +245,51 @@ public void testFilters()
                 "FROM lineitem\n" +
                 "WHERE\n" +
                 "    shipinstruct = 'TAKE BACK RETURN' AND orderkey < 10");
+
+        // BooleanStreamReader
+        assertQuery(ariaSession(), "SELECT linenumber FROM lineitem_aria WHERE is_returned = TRUE",
+                "SELECT linenumber FROM lineitem WHERE returnflag = 'R'");
+
+        assertQuery(ariaSession(), "SELECT linenumber, is_returned FROM lineitem_aria WHERE is_returned = TRUE",
+                "SELECT linenumber, true FROM lineitem WHERE returnflag = 'R'");
+
+        // Filter on a float column
```

Comment:
Are there cases where one of these tests tests something that the below queries don't?  If so, could we write a quick comment about what that is?

Reply:
@tdcmeehan First query filters and projects a column of an interesting type. Second query filters but doesn't project that column. (e.g. outputChannel == -1)